processor/cumulativetodeltaprocessor/config.go

```diff
@@ -19,26 +19,50 @@ import (
 	"time"
 
 	"go.opentelemetry.io/collector/config"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal/processor/filterset"
 )
 
 // Config defines the configuration for the processor.
 type Config struct {
 	config.ProcessorSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
-	// List of cumulative metrics to convert to delta.
-	Metrics []string `mapstructure:"metrics"`
-
 	// MaxStaleness is the total time a state entry will live past the time it was last seen. Set to 0 to retain state indefinitely.
 	MaxStaleness time.Duration `mapstructure:"max_staleness"`
+
+	// Deprecated. List of cumulative metrics to convert to delta.
```

Comment:
I think this needs to be `Deprecated: ` (with the colon) to be picked up by linters (see [here](https://github.com/golang/go/wiki/Deprecated))
```suggestion
	// Deprecated: List of cumulative metrics to convert to delta.
```